### PR TITLE
Replace 3 Wasm proposals checks with a single

### PR DESCRIPTION
**Pull request by JSMonk**

### CODE REVIEW

File: shared/src/webMain/resources/index.html [hunk 1]
```diff
@@ -149,15 +149,36 @@
...
-
-            const hasWasmExceptionSupport = () =>
-                WebAssembly.validate(new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 4, 1, 96, 0, 0, 3, 2, 1, 0, 10, 8, 1, 6, 0, 6, 64, 25, 11, 11])
-                );
-
-            const hasWasmReferenceTypeSupport = () =>
-                WebAssembly.validate(new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 4, 1, 96, 0, 0, 3, 2, 1, 0, 10, 7, 1, 5, 0, 208, 112, 26, 11]))
+            /*
+            Represents the next Wasm module:
+            (module
```
Comment (bashor) [inline shared/src/webMain/resources/index.html:154]: A command line how to compile it also would be helpful.

**Review by bashor — APPROVED** [review]

File: shared/src/webMain/resources/index.html (hunk 1 above)
Comment (JSMonk) [reply shared/src/webMain/resources/index.html:155]: Done 👌

**Review by bashor — APPROVED** [review]

**Comment by JSMonk** [comment]
@bashor seems like those changes were removed from the `master` branch -_-
Someone force pushed the reop

**Comment by bashor** [comment]
@JSMonk Why do you think so? It seems on the head now https://github.com/JetBrains/kotlinconf-app/commit/f7a148eb148376d6b4b7a8f2b10187074c559d46

**Comment by JSMonk** [comment]
@bashor Oh, okay. Seems like a lag of GitHub, because I'd seen the old code in the master branch. Now, seems like, it's synced

**Pull request merged**